alica_engine/include/engine/AlicaContext.h

```diff
@@ -249,8 +269,20 @@ class AlicaContext
     std::unique_ptr<AlicaEngine> _engine;
     std::unique_ptr<IAlicaCommunication> _communicator;
     std::unordered_map<size_t, std::unique_ptr<ISolverBase>> _solvers;
+    std::unique_ptr<AlicaClock> _clock;
```

Comment:
Setting AlicaClock to a custom one should be optional. In AlicaContext constructor, please initialize it with default AlicaClock implementation.

Reply:
Okay, I will do that.